mammoth/utils/optimizers.py

```diff
@@ -83,55 +54,63 @@ def build_torch_optimizer(model, opts, task_queue_manager):
     Returns:
       A ``torch.optim.Optimizer`` instance.
```

Comment:
feels like we're missing a class constructor signature inspection + kwargs popping approach